README.md

```diff
@@ -124,6 +125,24 @@ JavaScript: # Enable TypeScript
 | Before | After |
 | --- | --- |
 | Beautified CSS files | ![There is no document formatter for 'css'-files installed.](https://user-images.githubusercontent.com/1885333/41637148-d28b5764-7428-11e8-9763-007f44fe6787.png) |
+</details>
+
+<details><summary>What can I do when a beautifier takes to much time formatting</summary>
+
+This section is for you if you are seeing the following error message in the developer console after running Format Document or similar:
+
+![Timeout message](docs/screenshot-timeout-message.png)
+
+Sometimes beautifiers are taking much time formatting your file, this can be caused by big files or many settings you have set for your beautifier, then you need to increase the VSCode __editor.formatOnSaveTimeout__ option to give the beautifier more time formatting your file. 
```

Comment:
I prefer to use `**` instead of `__` for bolding. Let's be consistent 😄.

```markdown
# Before
__editor.formatOnSaveTimeout__

# After
**editor.formatOnSaveTimeout**
```